### PR TITLE
GAWB-2014 bump spray version to dodge vuln

### PR DESCRIPTION
**Pull request by helgridly**
doge chosen by wheel

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [x] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell your tech lead (TL) that the PR exists if they want to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,11 +2,11 @@ import sbt._
 
 object Dependencies {
   val akkaV = "2.3.6"
-  val sprayV = "1.3.2"
+  val sprayV = "1.3.3"
   val slickV = "3.1.1"
 
   val googleDependencies = Seq(
-    "io.spray" %% "spray-json" % "1.3.2",
+    "io.spray" %% "spray-json" % sprayV,
```
Comment (rtitle) [inline project/Dependencies.scala:9]: There's another hardcoded spray version down below under `modelDependencies`:
```
"io.spray" %% "spray-json" % "1.3.2",
```

Comment (helgridly) [reply project/Dependencies.scala:9]: Great catch, thank you!

**Comment by rtitle** [comment]
This page lists 1.3.4 as the latest version:
http://spray.io/project-info/current-versions/

But looking in maven central the latest spray-json is 1.3.3, while spray-http and spray-httpx have 1.3.4 artifacts. Not sure why there's a discrepancy but 1.3.3 is probably fine.
👍

**Comment by helgridly** [comment]
Yeah, I just picked the version that the security tracker (in the ticket)
said fixed it.

File: project/Dependencies.scala (hunk 1 above)
Comment (jmthibault79) [reply project/Dependencies.scala:9]: FYI I'm planning to refactor this file soon, to help avoid such things

**Pull request merged**